node/nodeTransactionsEvents.go

```diff
@@ -0,0 +1,130 @@
+package node
+
+import (
+	"encoding/hex"
+
+	"github.com/ElrondNetwork/elrond-go/core/dblookupext"
+	"github.com/ElrondNetwork/elrond-go/data/receipt"
+	"github.com/ElrondNetwork/elrond-go/data/smartContractResult"
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+)
+
+func (n *Node) putEventsInTransaction(hash []byte, tx *transaction.ApiTransactionResult, epoch uint32) {
+	eventsHashes, err := n.historyRepository.GetEventsHashesByTxHash(hash, epoch)
+	if err != nil {
+		return
```

Comment:
I think we can let as it is because it's a pretty big chance this function to return an error because a lot of transaction will not have resultsHashes  (move balance transactions)

Reply:
Oh, I understand - currently, we cannot differentiate between "missing from storer" and "error when reading from storer".